src/ucp/api/ucp.h

```diff
@@ -1368,6 +1370,11 @@ typedef struct ucp_conn_request_attr {
      * server.
      */
     struct sockaddr_storage client_address;
+
+    /**
+     * User defined endpoint id of the remote client.
```

Comment:
User defined client id when connecting to remote socket address. This value is available via `@ref ucp_conn_request_query` function on the server side.